pkg/descheduler/descheduler.go

```diff
@@ -65,6 +65,10 @@ func Run(rs *options.DeschedulerServer) error {
 		return fmt.Errorf("deschedulerPolicy is nil")
 	}
 
+	for name, strategy := range deschedulerPolicy.Strategies {
```

Comment:
@damemi I wonder if it would be more reasonable to print entire configuration? As is done in the kube-scheduler case.

Reply:
The kube-scheduler does a lot of defaulting, so it was more helpful to print out the config there because you can't easily tell what is running just by looking at your own config. We don't have any default strategies in descheduler (users must set their own config that controls the whole descheduler), so I think it would be redundant to print the entire config